lib/recipe.rb

```diff
@@ -30,24 +31,26 @@
 
 # only support solaris 10 and 11
 return if platform?('solaris2') && node['platform_version'] !~ /^5.(10|11)$/
+
 liveness_agent = <<'AUTOMATE_LIVENESS_AGENT'
 #LIVENESS_AGENT
 AUTOMATE_LIVENESS_AGENT
 liveness_agent.gsub!('#!/usr/bin/env ruby', "#!#{Gem.ruby}")
 
-agent_dir         = Chef::Config.platform_specific_path(
```

Comment:
It did before mark added the case for windows. Since we're explicitly handling windows and non-windows paths we don't need it anymore. Before it was translating `/var/log/chef` to `c:/log/chef` on windows but that's a problem because we needed it to be `c:/chef/log`.